inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_sequences_to_sequences_ie.cpp

```diff
@@ -84,14 +82,12 @@ ngraph::pass::ConvertGRUSequenceMatcher::ConvertGRUSequenceMatcher() {
             return false;
         }
 
-        auto W = std::dynamic_pointer_cast<ngraph::opset4::Constant>(
-                gru_sequence->input_value(3).get_node_shared_ptr());
+        auto W = gru_sequence->input_value(3).get_node_shared_ptr();
         if (!W) {
```

Comment:
Looks like this checks are no more needed.

Reply:
removed